jobs/ci-kubernetes-e2e-kubeadm-gce.sh

```diff
@@ -26,9 +26,18 @@ readonly testinfra="$(dirname "${0}")/.."
 
 export PROJECT="k8s-jkns-e2e-kubeadm-gce-ci"
 export KUBERNETES_PROVIDER=kubernetes-anywhere
+
+# This job only runs against the kubernetes repo, and bootstrap.py leaves the
+# current working directory at the repository root. Grab the SCM_REVISION so we
+# can use the .debs built during the bazel-build job that should have already
+# succeeded.
+export SCM_VERSION=$(./hack/print-workspace-status.sh | grep ^STABLE_BUILD_SCM_REVISION | cut -d' ' -f2)
```

Comment:
Hrm, I don't think it can be in the image runner. The call stack looks like:

    ci-kubernetes-e2e-kubeadm/runner # image entrypoint
    \-> bootstrap.py
       \-> jobs/ci-kubernetes-e2e-kubeadm-gce.sh (the script in question)
          \-> e2e-runner.sh
             \-> kubetest

The kubernetes repo gets pulled by `bootstrap.py`, so the logic needs to be executed after that step, and I need to adjust the `E2E_OPTS` before `e2e-runner.sh` is invoked. I thought the job script was the perfect place for this, but if we only want to support static environment variables there, then does that mean I'll need to create a custom `scenario` to support this kind of logic (like kops has)?